cores/arduino/stm32/stm32_eeprom.h

```diff
@@ -56,18 +81,14 @@ extern "C" {
 #define FLASH_PAGE_SIZE     ((uint32_t)(16*1024)) /* 16kB page */
 #endif
 #define E2END (FLASH_PAGE_SIZE - 1)
+#endif
 
 /* Exported macro ------------------------------------------------------------*/
 /* Exported functions ------------------------------------------------------- */
 
 uint8_t eeprom_read_byte(const uint32_t pos);
 void eeprom_write_byte(uint32_t pos, uint8_t value);
 
-void eeprom_buffer_fill();
```

Comment:
Why remove those definition?

Reply:
I moved them to .c file: https://github.com/stm32duino/Arduino_Core_STM32/pull/717/files/136cf8f77a3aaae46aa18db590a983c6d1b6e16f#diff-6eec0914a305be509068695e0ded8d2fR44
Because those seem to be internal use only, as they are functions related to EEPROM/Flash memory buffer. I don't come up with any use case to expose the internal buffer and they are not safe to touch it directly. [EEPROM.h](https://github.com/stm32duino/Arduino_Core_STM32/blob/master/libraries/EEPROM/src/EEPROM.h) not using those functions supports my idea.
Also, RETRAM implementation uses VBAT-powered RAM directly so it doesn't need the buffer.

Reply:
No they can be used externally.
See: https://github.com/stm32duino/STM32Examples/blob/master/examples/NonReg/BufferedEEPROM/BufferedEEPROM.ino

Reply:
I didn't know about the examples repo, I will move them back.
I will use a buffer to reproduce the same result.